management/client/common/types.go

```diff
@@ -0,0 +1,19 @@
+package common
+
+// LoginFlag introduces additional login flags to the PKCE authorization request
+type LoginFlag uint8
+
+const (
+	// LoginFlagPrompt adds prompt=login to the authorization request
+	LoginFlagPrompt LoginFlag = iota
```

Comment:
I don't fully understand this part of the change.
The code means that `LoginFlagPrompt` has a numeric value of zero.
Which means that if the Management Server does not send any login flag then that would translate to `LoginFlagPrompt` .
Is that the intended behavior?  To have `LoginFlagPrompt` as a default?

Another question, then, is how to unset this flag?
Is setting `LoginFlagMaxAge0` the only way to not set `LoginFlagPrompt`?